config/sync/migrate_plus.migration.stanford_localist_importer.yml

```diff
@@ -494,6 +494,63 @@ process:
           entity_type: taxonomy_term
           ignore_case: true
           source: name
+  image_ext:
+    - plugin: skip_on_empty
+      method: process
+      source: photo_url
+    - plugin: explode
+      delimiter: .
+    - plugin: array_pop
+  image_destination_part_1:
+    - plugin: skip_on_empty
+      method: process
+      source: '@image_ext'
+    - plugin: concat
+      source:
+        - constants/image_path
+        - urlname
+  image_destination:
+    - plugin: skip_on_empty
+      method: process
+      source: '@image_destination_part_1'
+    - plugin: concat
+      source:
+        - '@image_destination_part_1'
+        - '@image_ext'
+      delimiter: .
+  image_file:
+    - plugin: skip_on_empty
+      method: process
+      source: '@image_destination'
+    - plugin: file_import
+      source: photo_url
+      destination: '@image_destination'
+      id_only: true
+  media_tag:
+    plugin: default_value
+    default_value: ' '
```

Comment:
No, that's not what I meant. we want the media tag, but you need to set the term name.